server/server.go

```diff
@@ -65,12 +73,8 @@ func ListenAndServe(ctx context.Context, httpsPort, httpPort int, handler http.H
 	}
 
 	var errorLog *log.Logger
-
 	if opts.IgnoreTLSHandshakeError {
-		debugWriter := &tlsHandshakeErrorWriter{
-			defaultWriter: writer,
-			debugWriter:   logrus.StandardLogger().WriterLevel(logrus.DebugLevel),
-		}
+		debugWriter := &TLSErrorDebugger{}
 		errorLog = log.New(debugWriter, "", log.LstdFlags)
```

Comment:
Nit: standard flags ends up creating duplicate timestamps, since standard logger adds a timestamp to the payload.

```golang
	l := log.New(&TLSErrorDebugger{}, "", log.LstdFlags)
	l.Print("test")
	l.Print("http: TLS handshake error: EOF")
```

```
time="2009-11-10T23:00:00Z" level=error msg="2009/11/10 23:00:00 test"
time="2009-11-10T23:00:00Z" level=error msg="2009/11/10 23:00:00 http: TLS handshake error: EOF"
```

Just use no flags:


```suggestion
		errorLog = log.New(&TLSErrorDebugger{}, "", 0)
```